### PR TITLE
Add taar-lite guid to guid rec ETL job to mozetl.

### PR DESCRIPTION
**Pull request by mlopatka**

### CODE REVIEW

**Comment by mlopatka** [comment]
@acmiyaguchi I made the requested changes to your previous review.
Test coverage is a WIP, but I think this is the direction that you were thinking.

**Comment by mlopatka** [comment]
@crankycoder can you take a look at the failing test assertion?

File: mozetl/taar/taar_lite_guidguid.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+             .builder
+             .appName("taar_lite=")
+             .enableHiveSupport()
+             .getOrCreate())
+
+    logging.info("Loading telemetry sample.")
+    longitudinal_addons = load_training_from_telemetry(spark)
+
+    # Only for logging, not used, but may be interesting for later analysis.
+    guid_set_unique = longitudinal_addons.withColumn("exploded", explode(longitudinal_addons.installed_addons)).select(
```
Comment (acmiyaguchi) [inline mozetl/taar/taar_lite_guidguid.py]: The sections that takes `longitudinal_addons -> addon_co_installations_collapsed` can be put into a separate transformation function. The test can then assert that the dataframe (or collected dataframe) looks right.

File: tests/test_taar_lite_guidguid.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+@mock.patch('mozetl.taar.taar_lite_guidguid.load_training_from_telemetry',
+            return_value=MOCK_TELEMETRY_SAMPLE)
+@mock_s3
+def test_load_training_from_telemetry(spark):
+    # Sanity check that mocking is happening correctly.
+    assert taar_lite_guidguid.load_training_from_telemetry(spark) == MOCK_TELEMETRY_SAMPLE
+
+# Exercise the only part of the ETL job happening outside of spark.
+def test_addon_keying():
+    assert taar_lite_guidguid.key_all(MOCK_KEYED_ADDONS) == MOCK_ADDON_INSTALLATIONS
```
Comment (acmiyaguchi) [inline tests/test_taar_lite_guidguid.py]: This is definitely in the right direction. You should break out the transformation in the main function and pass the mocked training data into it around here.

**Comment by acmiyaguchi** [comment]
Merging this into `mozilla:dev` for hand-off. Further work can be added as a PR onto this new branch (and can be merged into master afterwards.

**Pull request merged**